specreduce/tracing.py

```diff
@@ -302,15 +312,21 @@ def __post_init__(self):
 
         warn_bins = []
         for i in range(self.bins):
-            # repeat earlier steps to create gaussian fit for each bin
+
+            # binned column (sum) if bins < ncols. otherwise, just columns.
```

Comment:
can you elaborate more in this comment, this logic is hard to follow without really following where everything comes from above.